css/_list.scss

```diff
@@ -180,7 +180,7 @@ margin-left: 30px;
 
 	&:hover{
 		border-left: 3px solid $gray-500;
-		background-color:$gray-300;
+		background-color:$gray087;
```

Comment:
is it safe to use a gradient there? Don't we want the background on hover to be the same color all the time ?